electron-app/src/components/status/BalanceSavedIndicator.vue

```diff
@@ -24,12 +24,29 @@
           Never
         </span>
       </v-row>
+      <v-divider></v-divider>
+      <v-row>
+        <v-btn color="primary" outlined @click="refreshAllAndSave()">
+          <v-icon left>fa fa-save</v-icon>force save
+        </v-btn>
+        <v-tooltip bottom>
+          <template #activator="{ on }">
+            <v-icon class="ml-3" v-on="on">fa fa-info-circle</v-icon>
+          </template>
+          <div>
+            This will refresh all balances, ignore cache, and save your
```

Comment:
-> This will force taking a snapshot of all your balances, ignoring any cache.

Reply:
Done.